crates/networking/discv5/src/eth2.rs

```diff
@@ -0,0 +1,103 @@
+use alloy_primitives::{B256, aliases::B32, fixed_bytes};
+use ethereum_ssz::Encode;
+use ethereum_ssz_derive::{Decode, Encode};
+use ream_consensus::fork_data::ForkData;
+
+// Constants
+pub const FAR_FUTURE_EPOCH: u64 = 18446744073709551615;
+pub const GENESIS_FORK_VERSION: B32 = fixed_bytes!("0x00000000");
+pub const GENESIS_VALIDATORS_ROOT: B256 =
+    fixed_bytes!("0x0000000000000000000000000000000000000000000000000000000000000000");
+pub const ENR_ETH2_KEY: &str = "eth2";
+// Types
+#[derive(Default, Debug, Encode, Decode)]
+pub struct ENRForkID {
+    pub fork_digest: B32,
+    pub next_fork_version: B32,
+    pub next_fork_epoch: u64,
+}
+
+impl ENRForkID {
+    pub fn new(current_fork_version: B32, next_fork_version: B32, next_fork_epoch: u64) -> Self {
+        let fork_digest = ForkData {
+            current_version: current_fork_version,
+            genesis_validators_root: GENESIS_VALIDATORS_ROOT,
+        }
+        .compute_fork_digest();
+
+        Self {
+            fork_digest,
+            next_fork_version,
+            next_fork_epoch,
+        }
+    }
+
+    pub fn new_pectra() -> Self {
+        // Pectra fork version and epoch values
+        let current_fork_version = B32::from_slice(&[0x03, 0x00, 0x00, 0x00]); // This should be replaced with actual Pectra fork version
+        let next_fork_version = B32::from_slice(&[0x00, 0x00, 0x00, 0x00]); // FAR_FUTURE_EPOCH version
+        let next_fork_epoch = FAR_FUTURE_EPOCH;
+
+        Self::new(current_fork_version, next_fork_version, next_fork_epoch)
+    }
```

Comment:
```suggestion
    pub fn pectra() -> Self {
        // Pectra fork version and epoch values
        let current_fork_version = fixed_bytes!("0x05000000");
        let next_fork_version = current_fork_version; 
        let next_fork_epoch = FAR_FUTURE_EPOCH;

        let fork_digest = ForkData {
            current_version: current_fork_version,
            genesis_validators_root: GENESIS_VALIDATORS_ROOT,
        }
        .compute_fork_digest();

        Self {
            fork_digest,
            next_fork_version,
            next_fork_epoch,
        }
    }
```

The values for pectra are already defined here https://github.com/eth-clients/mainnet/blob/main/metadata/config.yaml

The `next_fork_version` should be `current_fork_version` in our case

We don't need the new function for now, this pectra function is sufficient